src/clients/youtube-api.js

```diff
@@ -0,0 +1,40 @@
+const baseAddress = 'https://youtube.googleapis.com/youtube/v3';
```

Comment:
For global constants, better use CAPITALIZED variable name in order to distinguish with local constants, in this case will be `BASE_ADDRESS`